Tasks/VsTest/vstest.ts

```diff
@@ -456,22 +704,25 @@ function resetRunInParallel() {
     runInParallel = false;
 }
 
+function compareNumbers(a,b) {
+    return a - b;
```

Comment:
got it. but require new function? .sort((a, b) => a - b);

---
In reply to: [68179944](https://github.com/Microsoft/vsts-tasks/pull/1932#discussion_r68179944) [](ancestors = 68179944)